tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -1284,4 +1336,31 @@ pub(super) mod test {
         assert_eq!(fmt(123456789012), "123s");
         assert_eq!(fmt(1234567890123), "1235s");
     }
+
+    #[test]
+    fn fmt_span_combinations() {
```

Comment:
TIOLI: it could be nice to have a test that we actually do reasonable behavior with various bitflag combinations, as well, but I'm not going to block merging this on that, the code is pretty straightforward.